lib/UserBackend.php

```diff
@@ -455,6 +455,11 @@ public function updateAttributes($uid,
 		} catch (\InvalidArgumentException $e) {
 			$newDisplayname = null;
 		}
+		try {
+			$newQuota = $this->getAttributeValue('saml-attribute-mapping-quota_mapping', $attributes);
```

Comment:
If the parameter we are looking for doesn't exists, this function returns '' (empty string). setQuota() will translate it in ' B'. I think this doesn't make sense. We should translate a empty string either to `null` so that nothing happens,  'default' or 'none'. Not sure what's the best in this case, but everything is probably better than setting the quota to ' B' if the parameter doesn't exists

Reply:
yes agreed. I would vote if none is returned we set it to 'default'. as that makes the most sense in my opinion.

Reply:
yes, this was also my initial thought... so let's go with 'default'